src/backend/distributed/test/run_from_same_connection.c

```diff
@@ -17,10 +17,10 @@
 
 #include "access/xact.h"
 #include "distributed/connection_management.h"
+#include "distributed/coordinator_protocol.h"
```

Comment:
during the iterations I removed and added some header files.

this line moved from below when I sorted the lines alphabetically. seems sane to keep it.